package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "rf-chrome-extension",
```

Comment:
it doesn't really matter what the name is I suppose. Since we're not publishing the package on npm